cogs/server_specific.py

```diff
@@ -34,6 +34,20 @@ async def dj_help(self, ctx):
         docstring = self.bot.remove_indentation(docstring)
         await ctx.send(docstring)
 
+    @commands.command(name="asight.help")
+    async def dj_help(self, ctx):
```

Comment:
This function needs to be renamed to something else, or it will overwrite the existing dj_help command.